v1/nix/modules/drv-parts/nodejs-granular/interface.nix

```diff
@@ -33,7 +33,7 @@ in {
         Whether to run a package's build script (aka. `npm run build`)
       '';
     };
-    nodejsDeps = {
+    deps = {
```

Comment:
While it's also including an extra level for package versions, conceptually it's the same as `pip.drvs`, right? Maybe we should call it like that?

Reply:
Regarding the naming, I agree let's consolidate these. Though I think just `deps` is a much better name than `drvs`. I'd prefer to do this in another PR.

Regarding the structure. We cannot have the same as in python, because in nodejs we can have several versions of the same dependency under `deps`.

Reply:
Yes, structure can't obviously be the same. Not sure about `deps` though, as that could cause confusion between e.g. `config.deps` and `config.pip.deps`?